nl_server/loader.py

```diff
@@ -166,3 +186,31 @@ def get_env_path(flask_env: str, file_name: str) -> str:
 
 def _is_custom_dc_dev(flask_env: str) -> bool:
   return flask_env == 'custom_dev'
+
+
+# Get embedding index information for an instance
+#
+def _get_embeddings_spec() -> EmbeddingsSpec:
+  embeddings_spec_dict = None
+
+  # If custom dc, get from constant
+  if is_custom_dc():
+    embeddings_spec_dict = _CUSTOM_DC_EMBEDDINGS_SPEC
+  # otherwise try to get from gke.
+  elif os.path.exists(_EMBEDDINGS_SPEC_PATH):
+    with open(_EMBEDDINGS_SPEC_PATH) as f:
+      embeddings_spec_dict = json.load(f) or {}
+  # If that path doesn't exist, assume we are running locally and use the values
+  # from autopush.
+  else:
+    current_file_path = Path(__file__)
+    autopush_env_values = f'{current_file_path.parent.parent}/deploy/helm_charts/envs/autopush.yaml'
+    with open(autopush_env_values) as f:
+      autopush_env = yaml.full_load(f)
+      embeddings_spec_dict = autopush_env['nl']['embeddingsSpec']
+
+  return EmbeddingsSpec(
+      embeddings_spec_dict['defaultIndex'],
+      embeddings_spec_dict['enabledIndexes'],
+      vertex_ai_model_info=embeddings_spec_dict.get('vertexAIModels') or {},
```

Comment:
nit: why not dict.get('vertexAIModels', {})

Reply:
when coming from the environment, the key will exist but value will be None and in both cases where key doesn't exist OR value is none, i want to use {}

Reply:
comment please :)